packages/lesswrong/lib/collections/clientIds/schema.ts

```diff
@@ -4,8 +4,6 @@ const schema: SchemaType<"ClientIds"> = {
   clientId: {
     type: String,
     canRead: ['sunshineRegiment','admins'],
-    /** Not actually nullable, but attempting to add the NOT NULL constraint causes the table to lock */
-    nullable: true
```

Comment:
Running `yarn generate` changed some of these in the fragment types to `string | null`, which caused some type errors. This is a bug in the fragment generation code, as omitting this field means the column is nullable from the database's perspective. But, I didn't want to fix this for this PR, and in this case client ids are actually never null and it's [just because of the size of the table that we can't add the contstraint](https://github.com/ForumMagnum/ForumMagnum/pull/9296)